src/data/customers.json

```diff
@@ -0,0 +1,9 @@
+[{"id":1,"customer-type":"particular","email":"vgiovannacci0@mozilla.com","buildings":1,"fiscal-address":"0269 Lunder Road"},
```

Comment:
buildings shouldn't be a list instead of just a number?

Reply:
Buildings data modified to an array. Thank you!